core/test/functional/api/v0.1/redirects_spec.js

```diff
@@ -10,30 +10,25 @@ var should = require('should'),
     ghost = testUtils.startGhost,
     request, accesstoken;
 
-should.equal(true, true);
```

Comment:
Done this small cleanup before copying the suite over to v2, to at least keep v2 tests more up to date :)